Casks/dbeaver-enterprise.rb

```diff
@@ -3,10 +3,14 @@
   sha256 "75e778c353d07fa02ed845fbb4f04afaa8d0462c46ac6b8e3a062349935fd873"
 
   url "https://dbeaver.com/files/#{version}/dbeaver-ee-#{version}-macos.dmg"
-  appcast "https://dbeaver.com/product/version.xml"
   name "DBeaver Enterprise Edition"
   homepage "https://dbeaver.com/"
 
+  livecheck do
+    url "https://dbeaver.com/product/version.xml"
+    regex(%r{<number>(\d+(?:\.\d+)*)</number>}i)
```

Comment:
```suggestion
    regex(%r{<number[^>]*?>v?(\d+(?:\.\d+)+)</number>}i)
```

Though this element doesn't have any attributes, it wouldn't hurt to defensively account for them (to prevent this from breaking if it happens).